packages/story-editor/src/components/fontCheck/fontCheckDialog.js

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { TranslateWithMarkup, __ } from '@googleforcreators/i18n';
+import { trackClick } from '@googleforcreators/tracking';
+import styled from 'styled-components';
+import { useState, useCallback, useMemo } from '@googleforcreators/react';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  Text,
+  THEME_CONSTANTS,
+  Icons,
+  Link,
+} from '@googleforcreators/design-system';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../app';
+import Dialog from '../dialog';
+import FontPicker from '../panels/design/textStyle/fontPicker';
+import { useConfig } from '../../app/config';
+import { TEXT_ELEMENT_DEFAULT_FONT } from '../../app/font/defaultFonts';
+
+const Spacer = styled.div`
+  margin-top: 22px !important;
+`;
+
+const ReversedIcon = styled(Icons.ChevronRightSmall)`
+  transform: rotate(180deg);
+`;
+
+const DashboardLink = styled.span`
+  display: flex;
+  color: black;
+  align-self: flex-start;
+  align-items: center;
+  font-weight: normal;
+  margin-right: 40px;
+`;
+
+export const FontCheckDialog = ({ isOpen, onClose, missingFont }) => {
+  const { dashboardLink, dashboardSettingsLink } = useConfig();
+  const defaultFont = useMemo(() => {
```

Comment:
nit: memoizing this actually doesn't have any true benefit. https://kentcdodds.com/blog/usememo-and-usecallback#how-is-usememo-different-but-similar

Reply:
useMemo was added here to cover a lint error 

Thoughts?

```
62:9   error  The 'defaultFont' object makes the dependencies of useCallback Hook (at line 99) change on every render. To fix this, wrap the initialization of 'defaultFont' in its own useMemo() Hook  react-hooks/exhaustive-dep
```

Reply:
You can move `defaultFont` outside the function component to fix this.